### DISCUSSION TITLE
ParaLink cannot display RING balance for the Hydration

### DISCUSSION THREAD

**Discussion by boundless-forest** [ringecosystem/discussion#5] [General]
From community member:

![image](https://github.com/user-attachments/assets/c228f7de-3a87-4d77-8075-03892de6749e)

---

![image](https://github.com/user-attachments/assets/8bb5b0c1-721d-490f-9406-1f89b4b3eed8)
![image](https://github.com/user-attachments/assets/5c6c3184-ea60-42db-9673-9c8845afcc12)

@hamidroohi92 @jiguantong  Please take a look.

**Comment by jiguantong** [comment]
![image](https://github.com/user-attachments/assets/6f6822bd-7ad7-4cb8-93de-5d4eb6a7c3d9)

The front end got the wrong token.
### Should get RING balance from tokens pallet.
![image](https://github.com/user-attachments/assets/bc07881d-40e9-4be3-a4c1-9c3d95e3ef02)

**Comment by hackfisher** [comment]
Created a issue for this discussion:

https://github.com/ringecosystem/paralink-ui/issues/46
